### ISSUE TITLE
Look and feel

### ISSUE THREAD

**Issue by pierre-24** [zestedesavoir/latex-template#7]
Where we discuss how pdf document should looks:

+ Colored links ?
+ Colored titles ?
+ Title of the current section on top of each page ? Author name on each page ?
+ Margins ?
+ Black and white option ?

**Comment by AmarOk1412** [comment]
+ Colored links? +1
+ Colored titles? I think it's perfect here.
+ Title of the current section on top of each page? +1 at the left
+ Title of the document on each page? +1 at the right
+ Author name on each page? -1
+ Margins? I think it's nice now.
+ Black and white option? yup
+ Printing option? No idea for now, we'll see

**Comment by Karnaj** [comment]
Good for color, we can change them easily.

**Comment by pierre-24** [comment]
> Black and white option? yup

Indeed. But then, if this option is activated, should we change every color back to black (or shades of grey) ?

**Comment by pierre-24** [comment]
The question remains :-°

**Comment by AmarOk1412** [comment]
woops.

Shades of gray are cool, but we should fix a limit (70% of black for text and 50% for other things (lines for example))
![2017-06-09-23 45 33-screenshot](https://user-images.githubusercontent.com/1888628/26999686-c17d6b5c-4d6d-11e7-9ba8-deb9a26e5e10.png)

**Comment by Karnaj** [comment]
> Shades of gray are cool, but we should fix a limit (70% of black for text and 50% for other things (lines for example))

Why not. Do you have an example (colors definition for some elements)?

**Comment by AmarOk1412** [comment]
> Do you have an example (colors definition for some elements)?
No. I didn't try anything yet.

**Comment by Heziode** [comment]
Maybe use another package for block (question, etc) for prevent "page overflow" and code issue.

I work on new "system" for fix this.

**Comment by Heziode** [comment]
Hi,

For block (question, etc.) I offer this :
![zds_block_1](https://user-images.githubusercontent.com/16735076/31093194-36b985c0-a7b1-11e7-8640-882956b5d44e.png)
![zds_block_2](https://user-images.githubusercontent.com/16735076/31093196-36be57da-a7b1-11e7-97f1-d88934badaa3.png)
![zds_block_3](https://user-images.githubusercontent.com/16735076/31093195-36bddcec-a7b1-11e7-8437-d03efc89b974.png)
![zds_block_4](https://user-images.githubusercontent.com/16735076/31093197-36c0c15a-a7b1-11e7-8d9b-c894973eed03.png)
![zds_block_5](https://user-images.githubusercontent.com/16735076/31093198-36c6642a-a7b1-11e7-9463-b2e521099c6d.png)

[PDF Link](https://github.com/zestedesavoir/latex-template/files/1349907/ZdS_block.pdf)

What I do :

- Information Block
- Question Block
- Warning Block
- Error / Danger Block
- Color for external link (+ add icon)
- Code environment : change environment for prevent display issue, adding background color, changing highlight color

*Note : All icon it's generated with tikz, no image added.*


What remains for me to do :
- SpoilerEnv
- Quotation Block
- Maybe a « success state block » in green ?
- What else ?

I await your feedback before continuing (to avoid continuing if it does not like)

**Comment by pierre-24** [comment]
That really looks good, so personally, I would be very pleased to see that in a pull request :) 

About the spoiler environment, please check #65, because I don't want this to break again ^^

> Color for external link (+ add icon)

Do you have a screenshot of that ? (please remember that the `print` option change the behavior, btw)

> Note : All icon it's generated with tikz, no image added.

Something that I personally likes ! <3 (but I don't now what are the dependencies of Tikz, so you'll probably have fun)

> Maybe a « success state block » in green ?

You can, since it is basically painless to add, but since it have no equivalent in ZMd right now, it would not be used.

> What else ?

I don't know whether it would be easy to go black and white (in fact, more "50 shades of grey") if the `print` option is activated, so that the code blocks, for example, would not drain the black quadridge. If it is not easy, don't bother, though ;)

**Comment by Heziode** [comment]
> Do you have a screenshot of that ? (please remember that the `print` option change the behavior, btw)

At the end of document : `DuckDuckGo est un moteur…`

Ok for spoiler I will check that.

**Comment by pierre-24** [comment]
Oh, indeed. Nice, then :)

**Comment by Karnaj** [comment]
**@Heziode** : [To know if you continue](https://zestedesavoir.com/forums/sujet/8250/zmarkdown/?page=5#p163545).

**Comment by Heziode** [comment]
Hi,

I've updated my work:

![zds_block_6](https://user-images.githubusercontent.com/16735076/31199270-20e2da06-a957-11e7-8076-9f054cda3771.png)

Update : 
- highlight of code
- border of code block

Add :
- Success Box
- Quotation Box
- Spoiler Box

For iframe I don't have idea at the moment

**Comment by pierre-24** [comment]
Still fan.

> For iframe I don't have idea at the moment.

What about more or less what you did for the spoilers, but with an icon [like this one](https://cdn4.iconfinder.com/data/icons/web-links/512/41-512.png) ?

**Comment by artragis** [comment]
Like I said, there is no need of dealing with iframe we do not integrate them to the tex code.

**Comment by pierre-24** [comment]
Yep, @Heziode remind me that by PM ;)

**Comment by Heziode** [comment]
Last update before PR (probably) :

![zds_cover](https://user-images.githubusercontent.com/16735076/31268768-1e5dae82-aa7e-11e7-802f-3b6246665c43.png)

![zds_block_7](https://user-images.githubusercontent.com/16735076/31268767-1e53bc9c-aa7e-11e7-9a29-29fae6e2fff0.png)


I have change color (and width) of horizontale line, change color of question block. And little change on cover.

**Comment by Heziode** [comment]
Hi,

For cover I purpose that:

![latex-template-cover](https://user-images.githubusercontent.com/16735076/31579086-8c49b696-b12e-11e7-9ef6-e208d34e54cd.png)

- When you click on author pseudo, you are redirected on profile of this user on ZestDeSavoir.com
- When you click on tuto icon, you are redirected on tuto on ZestDeSavoir.com
- When you click on licence, you are redirected on CreativeCommon legal code of this licence
- When you click on ZdS logo, you are redirected on main page of ZdS

**Comment by Heziode** [comment]
Update, I put the title a little lower:

![latex-template-cover](https://user-images.githubusercontent.com/16735076/31584294-0f518f00-b1ac-11e7-9e18-1adfaa6ff69f.png)

**Comment by Heziode** [comment]
Another Update:

![latex-template-cover-3](https://user-images.githubusercontent.com/16735076/31587167-ee0f66c4-b1dc-11e7-88c7-3720ef2ee182.png)

- Add simple date
- Decrease size of licence

**Comment by Heziode** [comment]
For replacement of default main LaTeX font I propose Noto: [test_main-font-noto.pdf](https://github.com/zestedesavoir/latex-template/files/1464833/test_main-font-noto.pdf)

**Comment by pierre-24** [comment]
Maybe it is me, but this font seems larger than the default one. Maybe it is better, but that's the only comment I can do right now ;)

**Comment by Heziode** [comment]
Indeed, it seems bigger than default font.
I had some additional feedback on the [forum](https://zestedesavoir.com/forums/sujet/8250/zmarkdown/?page=6#p166372) (and IRC).

**Comment by pierre-24** [comment]
Let's say we a good here. Our generated documents rocks <3
